geanydoc/src/geanydoc.c

```diff
@@ -284,15 +284,14 @@ on_comboboxType_changed(GtkComboBox * combobox, G_GNUC_UNUSED gpointer user_data
 
 #define GLADE_HOOKUP_OBJECT(component,widget,name) \
   g_object_set_data_full (G_OBJECT (component), name, \
-    gtk_widget_ref (widget), (GDestroyNotify) gtk_widget_unref)
+    g_object_ref (widget), (GDestroyNotify) gtk_widget_destroy)
```

Comment:
`gtk_widget_destroy` isn't the same as `gtk_widget_unref`, `g_object_unref` would be a lot closer.  It may or may not change anything, depending on how it's used, but it probably would be "safer" to use `g_object_unref`